### DISCUSSION TITLE
Locally linked plugins

### DISCUSSION THREAD

**Discussion by mlynch** [ionic-team/capacitor#419] [Ideas]
Verify npm linking plugin works and explain this in plugin dev guide

**Comment by jcesarmobile** [comment]
At the moment linked plugins are not detected because we only read the dependencies from package.json.
Do we want to detect the linked plugins somehow? or document that they have to add the plugin name inside the dependencies of the package.json

**Reply by imhoffd** [reply]
I believe this might be fixed by this PR: https://github.com/ionic-team/capacitor/pull/3040

**Comment by dotNetkow** [comment]
Documented this in the [Capacitor docs](https://capacitorjs.com/docs/plugins/workflow#local-testing).
